{
    "type": "bundle",
    "id": "bundle--131b2d80-bcb3-4297-a692-f662825cf274",
    "objects": [
        {
            "type": "vulnerability",
            "spec_version": "2.1",
            "id": "vulnerability--fc6141b1-4cbb-40fb-836b-13b8e7b33346",
            "created_by_ref": "identity--8ce3f695-d5a4-4dc8-9e93-a65af453a31a",
            "created": "2025-02-27T00:20:15.991797Z",
            "modified": "2025-02-27T00:20:15.991797Z",
            "name": "CVE-2022-49497",
            "description": "In the Linux kernel, the following vulnerability has been resolved:\n\nnet: remove two BUG() from skb_checksum_help()\n\nI have a syzbot report that managed to get a crash in skb_checksum_help()\n\nIf syzbot can trigger these BUG(), it makes sense to replace\nthem with more friendly WARN_ON_ONCE() since skb_checksum_help()\ncan instead return an error code.\n\nNote that syzbot will still crash there, until real bug is fixed.",
            "external_references": [
                {
                    "source_name": "cve",
                    "external_id": "CVE-2022-49497"
                }
            ]
        }
    ]
}